mol_def.py

```diff
@@ -38,6 +38,69 @@
 AMINO_ACID_DICT["CYSP"] = "C"
 
 
+NUCLEIC_ACIDS_DNA = {"DA": 'A',
+                     "DT": 'T', 
+                     "DC": 'C', 
+                     "DG": 'G', 
+
+                     "DA5": 'A',
+                     "DA3": 'A',
+                     "DT5": 'T', 
+                     "DT3": 'T', 
+                     "DC5": 'C', 
+                     "DC3": 'C', 
+                     "DG5": 'G', 
+                     "DG3": 'G', 
+                    }
+
+
+NUCLEIC_ACIDS_RNA ={"RA": 'A', 
+                    "RU": 'U', 
+                    "RC": 'C', 
+                    "RG": 'G', 
+
+                    "RA5": 'A', 
+                    "RA3": 'A', 
+                    "RU5": 'U', 
+                    "RU3": 'U', 
+                    "RC5": 'C', 
+                    "RC3": 'C', 
+                    "RG5": 'G', 
+                    "RG3": 'G', 
+                    }
+
+
+NUCLEIC_ACIDS = {"DA": 'A',
```

Comment:
Maybe there is no need to make the difference between DNA and RNA? The generic term `nucleic_acid` could be sufficient. However, it could be a good idea to keep to both `NUCLEIC_ACIDS_DNA` and `NUCLEIC_ACIDS_RNA` dictionaries and to create `NUCLEIC_ACIDS` as the merge of them: `NUCLEIC_ACIDS =  NUCLEIC_ACIDS_DNA | NUCLEIC_ACIDS_RNA`

Reply:
why | and not & ?

Reply:
I don't know. Try it.

Reply:
with `&` there is an error (`unsupported operand type(s) for &: 'dict' and 'dict'`). 
But works with `|` :)